src/AbstractBuilder.php

```diff
@@ -57,6 +64,25 @@ public function setNamespace(string $namespace): void
 
     public function setValues(array $values)
     {
+        if ($this->config['generatorOptions']['target'] === ConfigOptions::MODE_PSR2) {
+            if ($this instanceof EnumBuilder) {
```

Comment:
Das sollte stattdessen im `EnumBuilder` in einer überschriebenen Methode passieren.

Reply:
Ist umgestellt in eine überschreibene setValues(). ist viel besser so. :)